### ISSUE TITLE
[RTL] BottomOverlay

### ISSUE THREAD

**Issue by Siemienik** [handsontable/handsontable#8761]
### Description

When I set `dir='rtl'` the bottom overlay displayed incorrectly on the left of the website. It should display on the right, below the table.

### Steps to reproduce

Following this [configuration](https://github.com/Siemienik/handsontable-storybooks/blob/10bd304f4ae389538f62abc136a1cb531f5a4acd/vanilla/configurations.js#L436-L440) 

```js
configurations.FixedRowsBottom = {
    data: Handsontable.helper.createSpreadsheetData(60, 10),
    licenseKey: "non-commercial-and-evaluation",
    fixedRowsBottom: 1
};
```
I got: 
![image](https://user-images.githubusercontent.com/2893099/135082205-ef95ea23-a5f5-45de-9537-2ab2ea97cfa8.png)

**Comment by aninde** [comment]
Fixed with v12.0.0 https://jsfiddle.net/fedn304g/

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 and developed and done in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
